style.css

```diff
@@ -31,7 +35,30 @@
   background-image: url("./images/background.svg");
   background-size: 100vw;
   background-repeat: no-repeat;
-  overflow: hidden;
+}
+
+.menu-item-style {
+  background-image: url('./images/bluebg.png');
+  display: flex;
+  flex-wrap: wrap;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  height: 100vh;
+  width: 150%;
+  margin-top: -30px;
+  margin-right: 0;
```

Comment:
- [ ] The mobile menu according to the design specifications should overflow the page and cover the full-screen height and width. At the moment, the mobile menu sits at the top of the site and does not overflow the page when opened. Please make sure it follows the specifications of the design. 
- One of the steps I would advise is to remove the background image and use a background color property instead. Kindly look at the design below for reference

### Your Design
![image](https://user-images.githubusercontent.com/43995830/157404321-1bcc1727-30e9-4876-9ecd-fbb1e6b227f6.png)

![image](https://user-images.githubusercontent.com/43995830/157404416-fc17d96c-a690-4090-89db-938b8901ea17.png)


### Original Design
![image](https://user-images.githubusercontent.com/43995830/157404195-863c1de8-ff5b-45b7-92bb-18ed2cfa84f1.png)